### PR TITLE
add support for specifying a reviewer group

### PR DESCRIPTION
**Pull request by egonelbre**
This allows to limit the assignment to a specific gerrit group.

### CODE REVIEW

**Review by mniewrzal — APPROVED** [review]

File: src/main/java/io/storj/gerrit/plugins/codeowners/ReviewAssigner.java [hunk 1]
```diff
@@ -51,21 +54,46 @@ public class ReviewAssigner implements WorkInProgressStateChangedListener, Comme
...
-                if (accounts.size() > 0) {
-                    return accounts.get(0)._accountId;
+                final List<GroupInfo> groups = this.gerrit.groups().query(reviewerGroup).get();
+                if(groups.size() == 0) {
+                    log.error("failed to find reviewerGroup");
+                    return null;
                 }
+
+                for(GroupInfo group: groups) {
+                    if(group.members.contains(candidate)) {
```
Comment (elek) [inline src/main/java/io/storj/gerrit/plugins/codeowners/ReviewAssigner.java]: Don't we need a loop to check all the candidates? As far as I understand the `candidate` here is always `candidates.get(0)`, and if she or he doesn't part the group we don't check the next candidate...

Comment (egonelbre) [reply src/main/java/io/storj/gerrit/plugins/codeowners/ReviewAssigner.java]: I'm not sure whether there ever will be multiple candidates... theoretically we could, but I'm not sure whether there's any point.

Comment (elek) [reply src/main/java/io/storj/gerrit/plugins/codeowners/ReviewAssigner.java]: Oh, I see. I was misled by the `s/account/candidates/g` rename here. These are not candidates, this part is just retrieving the account information.

Candidates are handled below in `assign`. 

For me, the original `accounts` variable name was more expressive ;-)

**Review by elek — APPROVED** [review]

File: src/main/java/io/storj/gerrit/plugins/codeowners/ReviewAssigner.java (hunk 1 above)
Comment (egonelbre) [reply src/main/java/io/storj/gerrit/plugins/codeowners/ReviewAssigner.java]: Sure, I'll change it back.

**Pull request merged**